Makefile

```diff
@@ -122,47 +122,47 @@ lint: isort black codespell ruff pylint mypy
 
 
 .PHONY: isort
-isort: install-deps
+isort: generate-version install-deps
```

Comment:
Why do we need a generated version for this actions (not for building) ?

Reply:
`ch_tools/__init__.py` is a part of clickhouse tools source code and some linters don't work without this file.

```
$ gmake pylint
************* Module monrun_checks_keeper.keeper_commands
ch_tools/monrun_checks_keeper/keeper_commands.py:11:0: E0401: Unable to import 'ch_tools.common.clickhouse.config' (import-error)
ch_tools/monrun_checks_keeper/keeper_commands.py:12:0: E0401: Unable to import 'ch_tools.common.result' (import-error)
************* Module monrun_checks_keeper.main
ch_tools/monrun_checks_keeper/main.py:9:0: E0401: Unable to import 'ch_tools' (import-error)
ch_tools/monrun_checks_keeper/main.py:10:0: E0401: Unable to import 'ch_tools.common.cli.context_settings' (import-error)
ch_tools/monrun_checks_keeper/main.py:11:0: E0401: Unable to import 'ch_tools.common.cli.locale_resolver' (import-error)
ch_tools/monrun_checks_keeper/main.py:12:0: E0401: Unable to import 'ch_tools.common.result' (import-error)
ch_tools/monrun_checks_keeper/main.py:13:0: E0401: Unable to import 'ch_tools.monrun_checks_keeper.keeper_commands' (import-error)
ch_tools/monrun_checks_keeper/main.py:24:0: E0401: Unable to import 'ch_tools.monrun_checks_keeper.status' (import-error)
...
$ gmake mypy
ch_tools/common/clickhouse/config/__init__.py: error: Source file found twice under different module names: "common.clickhouse.config" and "ch_tools.common.clickhouse.config"
ch_tools/common/clickhouse/config/__init__.py: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#mapping-file-paths-to-modules for more info
ch_tools/common/clickhouse/config/__init__.py: note: Common resolutions include: a) adding `__init__.py` somewhere, b) using `--explicit-package-bases` or adjusting MYPYPATH
Found 1 error in 1 file (errors prevented further checking)
gmake: *** [Makefile:155: mypy] Error 2
```

I added `generate-version` as a dependency for all linters for consistency. Though I'm not sure what option is better - add the dependency for all linters or only for `mypy` and `pylint`.